recipe-server/normandy/recipes/management/commands/update_recipe_signatures.py

```diff
@@ -41,7 +41,7 @@ def handle(self, *args, force=False, **options):
         if count == 0:
             self.stdout.write('No disabled recipes to unsign')
         else:
-            self.stdout.write('Unsigning {} disabled recipes:'.format(count))
+            self.stdout.write(f'Unsigning {count} disabled recipes:')
```

Comment:
When a recipe is disabled (turned off for Firefox users), we delete the signature for it, so that it can't be executed easily from the API anymore.